### PR TITLE
Consistent JS dialogs and popups

### LINKED ISSUE #3546: Consistent JS dialogs and popups
Current dialogs for actions like deleting posts look different to the modal dialog's used for actions like adding reputation, which is odd.

Instead of this:

image

They should look like this: 

image

They should also be able to be edited via templates.

Related: https://github.com/mybb/mybb/pull/3439#issuecomment-418769179

### PR DESCRIPTION
**Pull request by zygzagZ**
Fixes #3546

Fixes https://github.com/mybb/mybb/pull/3439#issuecomment-418769179

Makes the delete post confirmation and other prompts look just like i.e. rep++ modal window.

I am not sure if this is the best way to send templates from PHP to JS (probably not), but I didn't come up yet with a better one. I could possibly copy the css.php from some other PR to merge JS, and there we could find a place for JS templates.

Instead of this:
image
Now it looks like this: 
image
And it is modifyable via templates

### CODE REVIEW

File: install/resources/mybb_theme.xml [hunk 1]
```diff
@@ -14093,12 +14107,12 @@ $(function() {
...
 		<template name="gobutton" version="120"><![CDATA[<input type="submit" class="button" value="{$lang->go}" />]]></template>
-		<template name="headerinclude" version="1820"><![CDATA[<link rel="alternate" type="application/rss+xml" title="{$lang->latest_threads} (RSS 2.0)" href="{$mybb->settings['bburl']}/syndication.php" />
+		<template name="headerinclude" version="1821"><![CDATA[<link rel="alternate" type="application/rss+xml" title="{$lang->latest_threads} (RSS 2.0)" href="{$mybb->settings['bburl']}/syndication.php" />
 <link rel="alternate" type="application/atom+xml" title="{$lang->latest_threads} (Atom 1.0)" href="{$mybb->settings['bburl']}/syndication.php?type=atom1.0" />
 <meta http-equiv="Content-Type" content="text/html; charset={$charset}" />
 <meta http-equiv="Content-Script-Type" content="text/javascript" />
 <script type="text/javascript" src="{$mybb->asset_url}/jscripts/jquery.js?ver=1813"></script>
-<script type="text/javascript" src="{$mybb->asset_url}/jscripts/jquery.plugins.min.js?ver=1813"></script>
+<script type="text/javascript" src="{$mybb->asset_url}/jscripts/jquery.plugins.js?ver=1821"></script>
```
Comment (zygzagZ) [inline install/resources/mybb_theme.xml]: Gotta change it back to .min.js

File: jscripts/jquery.plugins.min.js [hunk 2]
```diff
@@ -1 +1,67 @@
....defaults,s)).expires){var r=new Date;r.setMilliseconds(r.getMilliseconds()+864e5*s.expires),s.expires=r}s.expires=s.expires?s.expires.toUTCString():"";try{a=JSON.stringify(n),/^[\{\[]/.test(a)&&(n=a)}catch(t){}n=i.write?i.write(n,e):encodeURIComponent(String(n)).replace(/%(23|24|26|2B|3A|3C|3E|3D|2F|3F|40|5B|5D|5E|60|7B|7D|7C)/g,decodeURIComponent),e=(e=(e=encodeURIComponent(String(e))).replace(/%(23|24|26|2B|5E|60|7C)/g,decodeURIComponent)).replace(/[\(\)]/g,escape);var l="";for(var d in s)s[d]&&(l+="; "+d,!0!==s[d]&&(l+="="+s[d]));return document.cookie=e+"="+n+l}e||(a={});for(var c=document.cookie?document.cookie.split("; "):[],u=/(%[0-9A-Z]{2})+/g,p=0;p<c.length;p++){var f=c[p].split("="),h=f.slice(1).join("=");'"'===h.charAt(0)&&(h=h.slice(1,-1));try{var m=f[0].replace(u,decodeURIComponent);if(h=i.read?i.read(h,m):i(h,m)||h.replace(u,decodeURIComponent),this.json)try{h=JSON.parse(h)}catch(t){}if(e===m){a=h;break}e||(a[m]=h)}catch(t){}}return a}}return o.set=o,o.get=function(t){return o.call(o,t)},o.getJSON=function(){return o.apply({json:!0},[].slice.call(arguments))},o.defaults={},o.remove=function(e,i){o(e,"",t(i,{expires:-1}))},o.withConverter=e,o}return e(function(){})});
```
Comment (zygzagZ) [inline jscripts/jquery.plugins.min.js]: I figure this could be minified further, but what tool to use for it?

Comment (euantorano) [reply jscripts/jquery.plugins.min.js]: [`uglifyjs`](https://github.com/mishoo/UglifyJS2) is usually my choice.

**Comment by euantorano** [comment]
Looks good, I will need to do some testing.

Since you've already set the template versions and JS versions to `1821` we'll set this to the 1.8.21 milestone once we release 1.8.20.

**Comment by Eldenroot** [comment]
Good job!

Maybe would be nice to add a morr space between the buttons. Moreover why not merge it into 1.8.20, quite a small change just in design and 1.8.21 will be there not anytime soon. I just applied the code on my board and start testing.

**Comment by Eldenroot** [comment]
GUI looks mich better, I am using this with ajax removing for attachments. Works fine

**Comment by euantorano** [comment]
I'd like to get this merged for 1.8.20, just need to fix the conflict and change the template versions.

**Pull request merged**